src/pages/components/HistoryItem.qml

```diff
@@ -0,0 +1,91 @@
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import Sailfish.Browser 1.0
+
+ListItem {
+    id: root
+
+    property string search
+
+    function remove() {
+        view.model.remove(model.index)
+    }
+
+    function openLink() {
```

Comment:
I'd not expect opening to a new tab. Open in a new tab could be in the context menu.

This should be imo loading to the active tab by default. Lemme check.